src/main/java/org/apache/sysds/runtime/instructions/cp/StringObject.java

```diff
@@ -59,6 +59,11 @@ public String getStringValue(){
 	public Object getValue(){
 		return _value;
 	}
+	
+	@Override
+	public int getSize() {
+		return 36 + _value.length() * 2; //FIXME
```

Comment:
you can test this with JOL.

Reply:
Yeah, JOL is good. I thought of using java.lang.instrument.Instrumentation though, in a future commit.